include/quic/endpoint.hpp

```diff
@@ -119,7 +111,7 @@ namespace oxen::quic
 
             Path _path = Path{_local, remote};
 
-            net.call([&opts..., &p, path = _path, this]() mutable {
+            net.call([&opts..., &p, path = _path, this, remote_pk = std::move(remote.get_remote_key())]() mutable {
```

Comment:
This `move` won't do anything here, because it's being passed a const reference and it won't de-const it to be able to move it.  (Technically you get back a `const Type&&` which is a fairly useless reference type in practice).

What you *could* do, however, is add a second `get_remote_key()` overload that is `&&` qualified (and change the existing one from `const` to `const &`) so that you can do `remote_pk = std::move(remote).get_remote_key()` to be able to steal the value here, i.e.

```c++
        const ustring& get_remote_key() const & { return remote_pubkey; }
        ustring&& get_remote_key() && { return std::move(remote_pubkey); }
```

Reply:
(That is assuming that `remote` is never used after this capture, of course).